template-helpers/scan-results.js

```diff
@@ -41,6 +41,24 @@ function getScanResultsHeadline(args) {
   return args.fn(headlineStrings);
 }
 
+function getFacebookResultMessage(id, args) {
+  let message = getString(id, args);
+  message = message.replace("<span>", "<span class='bold'>");
+  let ctaHref = "";
+  switch (id) {
+    case "facebook-breach-what-to-do-1-headline":
+      ctaHref = "https://blog.mozilla.org/firefox/facebook-data-leak-explained/";
+      break;
+    case "facebook-breach-what-to-do-2-headline":
+      ctaHref = "https://blog.mozilla.org/firefox/mozilla-explains-sim-swapping/";
+      break;
+  }
+  message = message.replace("<a>", `<a target="_blank" href="${ctaHref}">`);
```

Comment:
Does this need a "noopener noreferrer" business?

Reply:
Otherwise, I think this all looks fine. Here's the 4 instances of `getFacebookResultMessage` hbs helper that I could easily spot:

```hbs
<p class="temp-marketing-p">{{{ getFacebookResultMessage "facebook-breach-note" }}}</p>
<h3 class="feature-title">{{{ getFacebookResultMessage "facebook-breach-what-to-do-1-headline" }}}</h3>
<h3 class="feature-title">{{{ getFacebookResultMessage "facebook-breach-what-to-do-2-headline" }}}</h3>
<a class="text-link blue-link recommendation" href="/breach-details/Facebook">{{{ getFacebookResultMessage "facebook-breach-what-to-do-3" }}}</a>
```

`facebook-breach-note` string doesn't include a link, so makes sense it isn't in the `switch` statement.

Both `facebook-breach-what-to-do-1-headline` and `facebook-breach-what-to-do-2-headline` have `<span>`s and `<a>`s
```yaml
facebook-breach-what-to-do-1-headline = <span>Set your information to “Only me” or other non-public setting in <a>your Facebook profile</a>.</span>

facebook-breach-what-to-do-2-headline = <span>Change the password, PIN, or other security credentials on your <a>mobile
  phone carrier accounts</a> to prevent SIM-swapping</span>.
```

`facebook-breach-what-to-do-3` seems to be the outlier, but it's just being used for the link text, which explains why it isn't caught by the `switch` statement above.
```yaml
facebook-breach-what-to-do-3 = See all of the recommendations on our Facebook leak page
```

Reply:
> Does this need a "noopener noreferrer" business?

I wasn't sure – since it points to our own(ed) pages?

Reply:
Yeah we typically don't put these on links to our own Mozilla pages. And leaving the referrer in place helps GA attribute traffic sources, so it's nice to keep them.